### DISCUSSION TITLE
Preparing audio data set

### DISCUSSION THREAD

**Discussion by sage-khan** [ludwig-ai/ludwig#1489] [Q&A] [answered]
I need to make a multilingual Automatic Speech recognition system. I have collected audio data set of about 200 GB. I want to know how to prepare/preprocess the audio data for training it offline on my system. It will be a speech to text as well as voice analyizer for speech emotion recognition. After training I want to implement it on live calls and on recorded as well. Can anyone help me on doing this ?

**Comment by ShreyaR** [comment, marked as answer]
Hi Sage, I'd recommend checking out the Ludwig documentation to get started on this.

Here's some relevant pages:
- [Preprocessing and training with an audio input feature](https://ludwig-ai.github.io/ludwig-docs/0.5/examples/speaker_verification/)
- [Generating text output features](https://ludwig-ai.github.io/ludwig-docs/0.5/examples/seq2seq/)
- [Serving Ludwig models via REST endpoints](https://ludwig-ai.github.io/ludwig-docs/0.5/user_guide/serving/#rest-endpoints)
